src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -24,6 +25,15 @@ public class PushToBlobFeed : MSBuild.Task
 
         public bool Overwrite { get; set; }
```

Comment:
From my recent testing in coreclr I think we also need an option to also create the container if it doesn't already exist. That will help in creating feeds on the fly.

Reply:
Ok, I'll add that. I thought we did not want to support automatically creating containers